docs/cql/cqlsh.rst

```diff
@@ -380,6 +380,9 @@ dumping all or portions of the schema.
 
 In any of the commands, ``DESC`` may be used in place of ``DESCRIBE``.
 
+You may also omit what you want to describe and just simply use ``DESCRIBE <name>``. This will look for the object 
+ in a specific order: keyspace, table, view, index, user-defined type, user-defined function, user-defined aggregate.
```

Comment:
A thought for the future (don't have to do it now): Maybe we should add a test for this order? E.g., pick a random name and create both a table and a type (for example) with this name and see that:
1. DESC xyz - finds the table (not the type)
2. DESC TYPE xyz - finds the type (and the table doesn't bother it).